content/guides/getting-started/writing-your-first-component-test.md

```diff
@@ -323,12 +323,13 @@ Now let's write some meaningful component tests.
 
 ## Testing components
 
-In order to test components, we're going to need to import two things into our
-spec file: a function to mount our component, and the component itself.
+In order to test components, we will use the custom
+[`cy.mount()`](/api/commands/mount) command that is scaffolded during
+[setup](/guides/getting-started/component-framework-configuration) to mount our
+components. Mounting the component will allow us to write tests against its
+rendered output.
 
-### Setting up imports
-
-::include{file=partials/import-mount-functions.md}
+### Importing the Component
```

Comment:
'Component' should be consistent in these two section titles (mounting below) should be consistent with capitalization.